### PR TITLE
docs(go-coding-style): Fix comments in the naked returns example

### PR DESCRIPTION
**Pull request by vikin91**
## Description

Addressing a comment from #7562 that arrived after merging.

## Checklist

Did none of the above due to the type of changes.

- [ ] Investigated and inspected CI test results
- [ ] Unit test and regression tests added
- [ ] Evaluated and added CHANGELOG entry if required
- [ ] Determined and documented upgrade steps
- [ ] Documented user facing changes (create PR based on [openshift/openshift-docs](https://github.com/openshift/openshift-docs) and merge into [rhacs-docs](https://github.com/openshift/openshift-docs/tree/rhacs-docs))

### Here I tell how I validated my change

- I read it again after a short break to minimize the bias

### Reminder for reviewers

In addition to reviewing code here, reviewers **must** also review testing and request further testing in case the
performed one does not seem sufficient. As a reviewer, you must not approve the change until you understand the
performed testing and you are satisfied with it.

### CODE REVIEW

File: .github/go-coding-style.md [hunk 1]
```diff
@@ -161,7 +164,7 @@ func split(sum int) (x, y int) {
 ```
 
 ```go
-// good: it is known what is being returned by looking only at the return statement
+// Good: it is clear what is returned by looking only at the return statement
 func split(sum int) (x, y int) {
```
Comment (rukletsov) [inline .github/go-coding-style.md:168]: I'm not sure about named return values and especially the fact that they show up in an example labeled as "good". In https://github.com/stackrox/stackrox/pull/7562#discussion_r1303108087 are you saying that this code snippet is reasonable?
```
func split(sum int) (x, y int) {
	if x % 2 == 0 {
		return x, x * x
	}
	x = sum * 4 / 9
	y = sum - x
	return x, y
}

**Review by rukletsov — COMMENTED** [review]
Thanks for the follow up!

File: .github/go-coding-style.md (hunk 1 above)
Comment (vikin91) [reply .github/go-coding-style.md:168]: Well, naked returns are only possible when using named return arguments. I personally do not have anything against named arguments for return and find your example fully okay, but I see in which direction you are pulling.

Are you saying that we should avoid named return arguments all the time, or this example isn't great to illustrate the problem of naked returns?

Comment (vikin91) [reply .github/go-coding-style.md:168]: >  before we had a clear statement "no naked returns".  Note that this statement says nothing about named return args.

I understand, but naked returns are possible only when using named args. I deducted from that, that named args are okay but not with naked returns.
It is good that we started this discussion as we will have more clarity now as to what shall be used and what not.

Comment (0x656b694d) [reply .github/go-coding-style.md:168]: Return parameters names may add additional hint to what is actually returned, if that's not possible/desirable to express in the function name.
Consider:
```go
type Figure interface {
	GetCenter() (x float, y float)
}
```

Comment (rukletsov) [reply .github/go-coding-style.md:168]: Perfect. Indeed, glad we spoke. Let me know once you make the change and I'll approve the PR.

**Comment by vikin91** [comment]
This PR has empty diff now, so I am closing it.

**Pull request closed without merging**